### DISCUSSION TITLE
Is it possible to somehow deploy .NET Aspire Apps locally?

### DISCUSSION THREAD

**Discussion by efonsecab** [microsoft/aspire#6812] [Q&A] [answered]
So, I have a customer who wants to run his web apps deployed locally, and I was asked if it would be possible to somehow deploy the .NET Aspire solutions locally.

**Comment by davidfowl** [comment]
There’s no concept of a “local deployment”.  The closest you can get today is using https://github.com/prom3theu5/aspirational-manifests and deploying to docker compose.

**Comment by davidfowl** [comment, marked as answer]
The closest to this is using the new docker compose publisher in 9.2:

https://learn.microsoft.com/en-us/dotnet/aspire/whats-new/dotnet-aspire-9.2#-deployment-improvements

**Reply by Levvy055** [reply]
However there is no Aspire dashoboard container in generated yaml file

**Reply by davidfowl** [reply]
Copy this https://github.com/davidfowl/aspire-ai-chat-demo/blob/main/AIChat.AppHost/DashboardExtensions.cs

**Reply by Levvy055** [reply]
WoW. Great. Thanks a lot. Now I only need to set up Nginx. It would be good to have something similar to that.

**Reply by davidfowl** [reply]
It takes one person to struggle with the configuration, figure it out and package it up as an aspire integration we can alll benefit from. I’d urge you to do that 😁.

PS: That same project has an example of using caddy to host the JavaScript app
